src/XIVLauncher.Common.Unix/Compatibility/WineRunner.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Serilog;
+using XIVLauncher.Common.Util;
+
+namespace XIVLauncher.Common.Unix.Compatibility;
+
+public class WineRunner
```

Comment:
Not sure if this needs to be called runner, when all it does is hold wine settings/configuration